### DISCUSSION TITLE
Civet Lang Support

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [vuejs/language-tools#5057] [Q&A] [answered]
Hello 👋 ,

I'm trying https://civet.dev TypeScript superset. It has a Vite plugin, and its working for *.civet files imported in a SFC as child components in Vue.
My only question now is how to allow `<script lang="civet">`, so that Civet can run in SFCs.

Thanks.

**Comment by pindatm** [comment, marked as answer]
https://github.com/vuejs/language-tools/tree/master/packages/language-plugin-pug is probably your best frame of reference to implement support
